libphal/phal_dump.C

```diff
@@ -427,15 +427,15 @@ void collectSBEDump(uint32_t id, uint32_t failingUnit,
 			    pdbg_target_path(proc), fapiRc);
```

Comment:
This is libphal related changes. update commit message "libphal: Fix SPR, XIR, GPR sequence in SBE dump "
Also for future reference add current speccail function register sequence ma nd chnages one.

Reply:
done

Reply:
Requested changes in the commit messages not on PR.

Reply:
Also do rebase in next refresh,

Reply:
fixed commit and rebasd

Reply:
Next time make sure run clang tool.